### PR TITLE
Akka 2.6.19

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Review by octonato — APPROVED** [review]
LGTM

**Review by jrudolph — APPROVED** [review]
LGTM

File: blog/_posts/2022-03-21-akka-2.6.19-released.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+short: We are excited to announce Akka 2.6.19
+category: news
+tags: [releases]
+---
+
+Dear hakkers,
+
+We are excited to announce a new patch release of Akka 2.6. Noteworthy additions and improvements in this release:
+
+ * Fix connection not re-establised post unreachability resolution, [#31081](https://github.com/akka/akka/issues/31081)
```
Comment (raboof) [inline blog/_posts/2022-03-21-akka-2.6.19-released.md:14]: perhaps call out the collaboration with https://github.com/nick-nachos?

**Review by raboof — APPROVED** [review]

File: blog/_posts/2022-03-21-akka-2.6.19-released.md (hunk 1 above)
Comment (patriknw) [reply blog/_posts/2022-03-21-akka-2.6.19-released.md:14]: good suggestion, added

**Pull request merged**